creation/web_base/condor_chirp

```diff
@@ -9,7 +9,12 @@
 
 if [ -f "$1" ]; then
     glidein_config="$1"
-    error_gen=$(grep '^ERROR_GEN_PATH ' "$glidein_config" | awk '{print $2}')
+
+    # import add_config_line function
+    add_config_line_source=$(grep -m1 '^ADD_CONFIG_LINE_SOURCE ' "$glidein_config" | cut -d ' ' -f 2-)
+    source "$add_config_line_source"
+
+    error_gen=$(gconfig_get ERROR_GEN_PATH "$glidein_config")
```

Comment:
prefer `.` to `source`
If it is only for ERROR_GEN_PATH you can also keep the grep (1 occurrence and that variable is safe - not modified by the scripts)

Reply:
reverted to using grep and added a comment line with the note about why grep is used still.